Source/WebCore/platform/EventTrackingRegions.cpp

```diff
@@ -28,7 +28,46 @@
 
 namespace WebCore {
 
-TrackingType EventTrackingRegions::trackingTypeForPoint(const String& eventName, const IntPoint& point)
+ASCIILiteral EventTrackingRegions::eventName(Event eventName)
+{
+    switch (eventName) {
+    case Event::MousedownEvent:
+        return "mousedown"_s;
+    case Event::MousemoveEvent:
+        return "mousemove"_s;
+    case Event::MouseupEvent:
+        return "mouseup"_s;
+    case Event::MousewheelEvent:
+        return "mousewheel"_s;
+    case Event::PointerdownEvent:
+        return "pointerdown"_s;
+    case Event::PointerenterEvent:
+        return "pointerenter"_s;
+    case Event::PointerleaveEvent:
+        return "pointerleave"_s;
+    case Event::PointermoveEvent:
+        return "pointermove"_s;
+    case Event::PointeroutEvent:
+        return "pointerout"_s;
+    case Event::PointeroverEvent:
+        return "pointerover"_s;
+    case Event::PointerupEvent:
+        return "pointerup"_s;
+    case Event::TouchendEvent:
+        return "touchend"_s;
+    case Event::TouchforcechangeEvent:
+        return "touchforcechange"_s;
+    case Event::TouchmoveEvent:
+        return "touchmove"_s;
+    case Event::TouchstartEvent:
+        return "touchstart"_s;
+    case Event::WheelEvent:
+        return "wheel"_s;
+    }
+    return ASCIILiteral();
+}
+
+TrackingType EventTrackingRegions::trackingTypeForPoint(Event eventName, const IntPoint& point)
```

Comment:
Let's also change the variable name from `eventName` to `event`.

Reply:
Changed.